### ISSUE TITLE
Suspend individual GPUs

### ISSUE THREAD

**Issue by davidpanderson** [BOINC/boinc#1459]
Add a feature for suspending individual GPUs
- manually
- when an exclusive app is running
- when computer is in use

This will involve both job scheduling and work fetch.
Eventually there should be a GUI; initially it could be done w/ config file.

**Comment by informatorius** [comment]
Is it currently possible automatically suspend all GPUs when an exclusive app is running?
Or would this feature include that?

**Comment by Cactose** [comment]
Yes!! I've needed this many times in the past. Very annoying to have to stop all GPU computation just because you need one GPU available for other things.

**Comment by ChristianBeer** [comment]
Related to #1149

**Comment by Artoria2e5** [comment]
Yes yes yes! I want to snooze the intel GPU for web browsing and the nvidia for heavyweight stuff like games.

**Comment by WeeToddDidd** [comment]
> Yes!! I've needed this many times in the past. Very annoying to have to stop all GPU computation just because you need one GPU available for other things.

Exactly, same situation here.  Its a shame that its a all or none snooze situation.

**Comment by cminnoy** [comment]
Great, really need it. Mostly interesting for me to control from boinccmd.

**Comment by tmanok** [comment]
Really tired to configuring the xml files, more configuration options like these need to be released. I'd also love to see GPU usage % and GPU time % just like CPU right now. My issue is that on a laptop with an intergrated graphics processing, I can limit the CPU usage to 25% and time to 25% but the CPU will still be 85-95C meaning my laptop is a jet engine. Meanwhile, my discrete NVIDIA RTX GPU is sitting there being useless and cold.
